### ISSUE TITLE
Changes Are Not Confirmed with a Status Message

### ISSUE THREAD

**Issue by MajoMaldonado** [equalizedigital/accessibility-checker#1757]
Location / URL: https://edac-vpat.instawp.co/wp-admin/admin.php?page=accessibility_checker_settings 
Screenshot: 

<img width="777" height="617" alt="Image" src="https://github.com/user-attachments/assets/b9f5b05d-ba41-4138-b0c7-084c6da7eb07" />

WCAG S.C : 4.1.3 Status Messages (Level AA) 

Issue Description: After activating the "Save Changes" button, the page refreshes without displaying or announcing a confirmation that the settings were saved successfully. As a result, screen reader users receive no feedback indicating that the action completed successfully. A programmatically identifiable status message should be provided so assistive technologies can announce the successful completion of the action.
